automate-collections-updates/automate_collections_updates.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+"""Update requirements.yml with latest collection versions."""
+
+import argparse
+import os
+import re
+import shutil
+import subprocess
+import sys
+import tempfile
+import yaml
+import tarfile
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--vendored_collections",
+        type=str,
+        default=os.environ.get("VENDORED_COLLECTIONS", "vendored_collections.yml"),
+        help="Path/filename for vendored_collections.yml",
+    )
+    parser.add_argument(
+        "--repository",
+        type=str,
+        default=os.environ.get("REPOSITORY", "rhel-system-roles"),
+        help="The repository to work on. Either rhel-system-roles or linux-system-roles",
+    )
+    args = parser.parse_args()
+
+    hsh = yaml.safe_load(open(args.vendored_collections))
+    collection_tarballs = {}
+
+    repo = args.repository
+    if repo == "rhel-system-roles":
+        repo_url = "https://src.osci.redhat.com:443/rpms/rhel-system-roles.git"
```

Comment:
should definitely not have this url in a public file - I would expect these settings will be provided in some sort of config file.  and the internal config file will only be stored internally somewhere